### PR TITLE
Add jaxb runtime support for java 11

### PR DESCRIPTION
**Pull request by yuzhu**

### CODE REVIEW

**Comment by yuzhu** [comment]
jenkins, test this with java 11 please

**Comment by yuzhu** [comment]
verified to have fixed java11 test error

File: core/server/common/pom.xml [hunk 1]
```diff
@@ -54,6 +54,12 @@
       <groupId>com.google.guava</groupId>
       <artifactId>guava</artifactId>
     </dependency>
+    <!--aws sdk requires jaxb binding at runtime-->
+    <dependency>
+      <groupId>com.sun.xml.bind</groupId>
+      <artifactId>jaxb-impl</artifactId>
+      <scope>runtime</scope>
+    </dependency>
```
Comment (ZacBlanco) [inline core/server/common/pom.xml:62]: I don't quite understand why this is required

Comment (yuzhu) [reply core/server/common/pom.xml:62]: https://stackoverflow.com/questions/52502189/java-11-package-javax-xml-bind-does-not-exist
Without it, java 11 will throw a runtime error.

**Comment by yuzhu** [comment]
alluxio-bot, merge this please

**Pull request merged**